community/resources/remote-doc-repo.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: full-width
+group: community
+title: Remote Doc Repos
+---
+
+To further expand documentation options for DevDocs, we offer a new opportunity for developers to contribute content through Community Engineering project repositories!
+
+**Remote Doc Repos** is a new process that pulls content from public GitHub repositories and generates it as part of DevDocs. The DevDocs team helps you add your content to the site. 
+
+{:.bs-callout .bs-callout-info}
+This is the first of many new options for expanding documentation for developers. We will add more information and examples soon.
+
+## Remote Doc Repo benefits
+
+ - Keep your documentation in the same repo as your code. 
+ - All content updates follow your repo pull request and issue processes.
```

Comment:
Consider:   Manage doc updates by following the same GitHub processes you use for your code